cves/kernel/CVE-2019-15807.yml

```diff
@@ -255,36 +21,100 @@ i18n:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
-sandbox:
-  question: |
-    Did this vulnerability violate a sandboxing feature that the system
-    provides?
+vccs:
```

Comment:
Looks like the curation tool didn't output this correctly so it's hurting everyone. The format of these should be something like:

vccs:
  - commit: (40-character hex string)
    note: Manually confirmed

Same for `fixes` and `interesting_commits` (if you have any)